src/SIL.XForge.Scripture/Services/NotesFormatter.cs

```diff
@@ -70,27 +72,28 @@ private static XElement FormatComment(Comment comment)
             XElement commentElem = new XElement("comment");
             commentElem.Add(new XAttribute("user", comment.User));
             commentElem.Add(new XAttribute("date", comment.Date));
-            if (comment.ExternalUser != null)
+            if (comment.ExternalUser is not null)
                 commentElem.Add(new XAttribute("extUser", comment.ExternalUser));
             if (comment.Deleted)
                 commentElem.Add(new XAttribute("deleted", "true"));
             commentElem.Add(FormatContent(comment.Contents));
             return commentElem;
         }
 
-        private static XElement FormatContent(XmlElement commentContents)
+        private static XElement FormatContent(XmlElement? commentContents)
         {
             XElement contentElem = new XElement("content");
-            if (commentContents == null)
-                contentElem.Add(new XElement("p", ""));
+            if (commentContents is null)
+                contentElem.Add(new XElement("p", string.Empty));
             else
             {
                 foreach (XmlNode node in commentContents.ChildNodes)
                     if (node.Name == "p")
                         contentElem.Add(FormatParagraph(node));
-                    else if (node.NodeType == XmlNodeType.Text)
+                    else if (node.NodeType == XmlNodeType.Text && node.Value is not null)
```

Comment:
This may be fine. I don't know if Paratext will ever care about `<content><p></p></content` vs string.Empty, but we can keep an eye out.

Reply:
XText required Value to not be null (via a compiler nullable warning), so this was really only to fix that warning. It should in practice never be null, as a text node should always have a value.